### ISSUE TITLE
Update model - change of num_classes

### ISSUE THREAD

**Issue by riyu94** [wuyifan18/DeepLog#21]
Hi Wufiyan

I am training data using your approach
The model works fine when tested, but gives some false positives for sequences not present in training dataset.
Hence, to improve accuracy, I am trying to load this pretrained model and train the model again with new data (update pretrained model with new dataset)
This retraining build a new model with new dataset with more num_classes, Hence I am not able to combine or append to the pretrained model.
Also, it doesn't seem to detect the sequences from the pretrained model (it overwrites the pretrained model)
What steps can I follow to use pretrained model as well as the new trained data for further testing.
Any help appreciated :)
